pkgs/applications/misc/terminal-colors/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, help2man, python3, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "terminal-colors";
+  version = "3.0.1";
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "eikenb";
+    repo = name;
+    rev = "v${version}";
+    hash = "sha256-hekt77/FhSTMEARVuck49/Q1dIuqkwbOYmgGD1IItyc=";
+  };
+
+  buildInputs = [ python3 ];
+  nativeBuildInputs = [ help2man ];
+
+  # This sed command modifies output of --version command in way that
+  # makes manpage generated by help2man(1) prettier.
+  postPatch = ''
+    sed -r -i "3s/([0-9.]+)/$name - \1\\n/" ./$name
+  '';
+
+  # Upstream shebang uses /usr/bin/env, which is not present in Nix
+  # sandbox, so we need to patch it before running help2man, otherwise
+  # it would fail with "no such file or directory".
+  buildPhase = ''
+    patchShebangs .
+    help2man -n 'display terminal colors' -N ./$name > $name.1
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $man/share/man/man1
+    install -m755 ./$name $out/bin
+    install -m644 ./$name.1 $man/share/man/man1
```

Comment:
```suggestion
    install -m755 ./$pname $out/bin
    installManPage $pname.1
```

Reply:
I do not thing it wins much. I still need to create directory for $out/bin.

Reply:
install creates it when you give it -D.

Reply:
Today I learned. Thanks. Adjusted.